scene/gui/popup.cpp

```diff
@@ -247,7 +247,9 @@ void PopupPanel::_update_child_rects() {
 
 		if (c == panel) {
 			c->set_position(Vector2());
-			c->set_size(get_size());
+			Vector2 panel_size = Vector2(get_size()) / this->get_content_scale_factor();
```

Comment:
```suggestion
			Vector2 panel_size = get_size() / this->get_content_scale_factor();
```
No need to cast `Vector2` to `Vector2`

Reply:
`get_size()` on `Window` returns `Size2i`, so the divide winds up truncating the float into an int. Is there a more elegant way to handle this?

Reply:
Oh yes my bad, thought it was a `Control`, this is the correct way 🙂

Reply:
Okay, good.